Terminal.Gui/Windows/Dialog.cs

```diff
@@ -119,6 +119,42 @@ public Dialog (ustring title, int width, int height, params Button [] buttons) :
 		/// </remarks>
 		public Dialog (ustring title, params Button [] buttons) : this (title: title, width: 0, height: 0, buttons: buttons) { }
 
+		/// <summary>
+		/// Initializes a new instance of the <see cref="Dialog"/> class using <see cref="LayoutStyle.Computed"/> positioning, 
+		/// with a <see cref="Border"/> and with an optional set of <see cref="Button"/>s to display
+		/// </summary>
+		/// <param name="title">Title for the dialog.</param>
+		/// <param name="border">The border.</param>
+		/// <param name="buttons">Optional buttons to lay out at the bottom of the dialog.</param>
+		public Dialog (ustring title, Border border, params Button [] buttons)
```

Comment:
I thought we didn't like parameterized constructors. Are these new constructors strictly needed? Couldn't users use the 'old' constructors and constructor initialization to get the job done?